odra-casper/test-vm/src/vm/casper_vm.rs

```diff
@@ -78,11 +99,108 @@ impl CasperVm {
         self.accounts[index]
     }
 
+    /// Gets the validator public key.
+    pub fn get_validator(&self, index: usize) -> PublicKey {
+        self.validators
+            .get(index)
+            .unwrap_or_else(|| panic!("Not enough validators: {}", index))
+            .public_key()
+    }
+
     /// Advances the block time by the specified time difference.
     pub fn advance_block_time(&mut self, time_diff: u64) {
         self.block_time += time_diff
     }
 
+    /// Advances the block time by the specified time difference and processes auctions
+    /// Giving the rewards to the validators
+    pub fn advance_with_auctions(&mut self, time_diff: u64) {
```

Comment:
Make a comment or even change the `time_diff`, so we know if these are seconds or millis.

Reply:
Fixed in f18044d